src/canisters/frontend/ic-asset/src/asset/config.rs

```diff
@@ -24,7 +25,7 @@ pub struct AssetConfig {
     pub(crate) allow_raw_access: Option<bool>,
 }
 
-pub(crate) type HeadersConfig = HashMap<String, String>;
+pub(crate) type HeadersConfig = BTreeMap<String, String>;
```

Comment:
no objections just curious, what triggered this change?

Reply:
there was a bug reported a few weeks ago where multiple duplicate principals were being added to the ACL. Adam fixed that bug by using a BTreeSet. I imagine this data structure provides similar guarantees of order and non-duplicate entries

Reply:
The operations need to be sortable, so that they appear in a specific order for evidence computation.  BTreeMap entries iterate in sorted order